design/ceph/osd-migration.md

```diff
@@ -40,135 +28,101 @@ considered in the future:
 ## Proposal
 
 ### API Changes
-- Add `spec.storage.store` in the Ceph cluster YAML.
+- Add `spec.storage.osdMigrate` in the Ceph cluster YAML.
 
   ```yaml
   storage:
-    store:
-      type: bluestore-rdr
-      updateStore: yes-really-update-store
+    osdMigrate:
+      event: UPDATE_STORE|ENABLE_ENCRYPTION
```

Comment:
What if the user wants to perform both the operation in the same migration?

Reply:
That's a good question. Technically it should be possible but we should not increase the complexity and let the users migrate the OSDs for one `event` at a time.

Reply:
But we can just make this api to take array of events?

Reply:
We are supporting both the operations. Users can migrate OSDs for both encryption and for changing OSD store type. Other options can also be added in the future.